src/framework/ui/uitypes.h

```diff
@@ -282,11 +283,7 @@ struct MenuItem : public UiAction
 
     QString shortcutsAsString() const
```

Comment:
May be better rename to something like `shortcutsTitle`

Reply:
renamed